maxspeed_signals.mss

```diff
@@ -392,7 +392,7 @@
       }
     }
   }
-  
+
```

Comment:
Time to clean this up, really. But I'd do it on all files at once, so expect that to be cleaned by an independent PR, on which you can then easily rebase.

Reply:
Good point, I will revert these changes for this PR. My IDE automatically formats files to remove trailing whitespace.